test/bridge/ActiveMediaListTest.cpp

```diff
@@ -18,17 +18,6 @@ namespace
 static const uint32_t defaultLastN = 2;
 static const uint32_t audioLastN = 3;
 
```

Comment:
This thread could grab all pending jobs and exit before we actually posted the StopJob on JobQueue. Then the ~JobQueue would stall forever. That happened in Jenkins all of a sudden. This test has been like this for over a year.